nservicebus/upgrades/5to6/index.md

```diff
@@ -10,20 +10,189 @@ related:
 ---
 
 
-## Move to .NET 4.5.2
+## Upgrading to NServiceBus 6
```

Comment:
this will already be inherited from the title. so remove and bump all the blow headings up one size